CHANGELOG.md

```diff
@@ -0,0 +1,334 @@
+# Carbon Mapper - Version Log
+
+Carbon Mapper is committed to continuously improving its scientific algorithms, applying updates to both future and historical data when necessary. This practice, known as “reprocessing,” is a widely used approach in scientific data management that ensures data remain accurate, consistent, and aligned with the latest scientific understanding, algorithms, and calibration standards. Reprocessing ensures consistency across data versions and sources by harmonizing datasets from different missions and instruments. This is crucial for creating seamless time series that support accurate analysis and monitoring of long-term trends.
+
+All updates to Carbon Mapper data are assigned a version number for each product level, with a detailed change log summarizing modifications and documenting the state of the corresponding algorithms. Each version reflects rigorously vetted algorithm improvements to enhance accuracy and reduce observation uncertainty. Our Algorithm Theoretical Basis Document contains algorithm justification for our most recent algorithms and citations with detailed overviews of our product levels and algorithms associated with each product level.
+
+We periodically reprocess historical data to ensure that past datasets benefit from the most recent, validated, and reliable algorithms. Previous versions of historical data remain accessible to users who may need them for specific applications or comparisons. Each historical reprocessing builds on the strengths of our existing algorithms, further enhancing data accuracy and refining precision to provide even more reliable emission estimates.
+
+* The Summary table has an overview of published data versions.
+* The Detailed Collection Impact Notes thoroughly explain how algorithms changed in a specified collection and what instruments and gases are impacted.
```

Comment:
Should this read "Detailed Version Notes" to align with the section title below?